precompiles/xtokens/src/lib.rs

```diff
@@ -138,17 +138,14 @@ where
 		let mut input = handle.read_input()?;
 
 		// Bound check
-		input.expect_arguments(2)?;
```

Comment:
Was this just wrong? (changed from `2` to `4`)

Reply:
In multiple places there were calls to check arguments, then parse input, then call again check arguments, then again parse input ...
However this will disapear completely soon as I'm adding a `read_args!` macro that will handle expect_arguments, parsing and revert annotations :)